packages/fxa-payments-server/src/routes/Product/PayPalButton/index.tsx

```diff
@@ -63,6 +63,7 @@ export const PaypalButton = ({
       const { token } = await apiGetPaypalCheckoutToken({ currencyCode });
       return token;
     } catch (error) {
+      error.code = 'general-paypal-error';
```

Comment:
Maybe get that string from lib/errors so we don't have the literal value in multiple places here?